common-api/src/main/java/org/pmiops/workbench/cdr/dao/DbDomainDao.java

```diff
@@ -0,0 +1,70 @@
+package org.pmiops.workbench.cdr.dao;
+import org.pmiops.workbench.cdr.model.DbDomain;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface DbDomainDao extends CrudRepository<DbDomain, Long> {
+
+    List<DbDomain> findByConceptIdNotNull();
+
+    DbDomain findByConceptId(long conceptId);
+
+    List<DbDomain> findByDbType(String db_type);
+
+    List<DbDomain> findByDbTypeAndConceptIdNot(String db_type,Long concept_id);
+
+        @Query(nativeQuery=true,value="select d.domain_id, d.domain_display, d.domain_desc,\n" +
```

Comment:
Note that I moved DbDomainDao and *also* slightly tweaked these queries in this PR, @biopete . (Let me know if you want me to split that up.)

In particular: we're now looking for standard_concept in ('S', 'C') when we were only searching for 'S' before (classifications can be in the data, too); and for findDomainTotals(), we're doing count(c.concept_id) instead of count(distinct c.concept_id), since the distinct is unneeded when only doing a single join.

Reply:
It turns out this (including 'C') slows down query results a bit. It feels like the right behavior, and matches our search results I think, but I guess there are a lot of C's out there to count. :)